src/lib/s3.js

```diff
@@ -0,0 +1,55 @@
+const { ClearResult } = require("./utils");
+const emptyBucket = async (bucketName, AWS) => {
+	const S3 = new AWS.S3();
+	const listResp = await S3.listObjectsV2({
```

Comment:
this is a paginated API, we need to recurse over it, unfortunately... just noticed we have the same issue in the `sls-remove` command too :-(